### DISCUSSION TITLE
Bills per whole hour or per n number of minutes ?

### DISCUSSION THREAD

**Discussion by anjanesh** [community/community#16773] [Codespaces] [answered]
I have a question regarding Codespaces billing.
I am on the free tier of GitHub but wouldn't mind paying for Codespaces if its billing per usage only.

If I code in for 2 hrs in a day (not necessarily 2 consecutive hours) - say 30 mins in the morning, 30 mins in the afternoon, 30 mins in the evening and 30 minutes in the night. Will it be counted as 2 hours or 4 hours ?

**Comment by dmilburn** [comment, marked as answer]
Hi @anjanesh, please see more information on codespaces billing [here](https://docs.github.com/en/billing/managing-billing-for-github-codespaces/about-billing-for-codespaces#about-billing-for-codespaces)

Codespaces are billed by both their compute and storage use. Both are per minute.

Also, please note that codespaces is currently only available for organizations but you can sign up for the individual beta here: https://github.com/features/codespaces/signup

**Reply by anjanesh** [reply]
This answers my question - thank you - I already got the codespaces beta email today - hence these questions.
